src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -22,7 +21,7 @@
 
     private final Optional<BibDatabase> database;
 
-    private final LayoutFormatter toUnicode = new LatexToUnicodeFormatter();
+    private final LatexToUnicodeFormatter toUnicode = new LatexToUnicodeFormatter();
```

Comment:
Not really. I had some issue with it and decided to go down the easiest path... (It doesn't implement LayoutFormatter anymore, do not know what I tried as an intermediate solution.)